agent/api/ecsclient/client.go

```diff
@@ -558,4 +558,4 @@ func (client *APIECSClient) GetResourceTags(resourceArn string) ([]*ecs.Tag, err
 		return nil, err
 	}
 	return output.Tags, nil
-}
+}
```

Comment:
nit: you removed the [extra line](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)